### PR TITLE
[docker] Change base image to ubuntu:20.04

### PR DESCRIPTION
**Pull request by WonyoungChoi**
For building EFL source, change base image of the `tizen-tools` docker image to `ubuntu:20.04` from `debian:buster-slim`.
It is necessary to build an EFL source directly for host build of the flutter-tizen engine.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -72,12 +72,16 @@ RUN ${SYSROOT_PATH}/build-rootfs.py --arch x86
...
 # Install packages for engine build.
 RUN apt-get update && \
     apt-get install -y git curl ca-certificates python && \
     apt-get clean
 
 # Copy tizen_tools from the previous stage.
-COPY --from=builder /home/user/tizen_tools/  /tizen_tools/
+COPY --from=builder /home/user/tizen-studio/tools/llvm-10/  /tizen_tools/toolchains/
+COPY --from=builder /home/user/tizen_tools/toolchains/  /tizen_tools/toolchains/
+COPY --from=builder /home/user/tizen_tools/sysroot/  /tizen_tools/sysroot/
```
Comment (WonyoungChoi) [inline Dockerfile:87]: Layers were distributed to improve image download speed.

**Comment by bwikbs** [comment]
@WonyoungChoi  Since you're working on docker image... I'm just asking..
Can you make an environment that xvfb-run can run in docker? (I mean we can run test shell in docker enviroment)
If so, it will be helpful for rendering tests.

**Comment by WonyoungChoi** [comment]
> @WonyoungChoi Since you're working on docker image... I'm just asking..
> Can you make an environment that xvfb-run can run in docker? (I mean we can run test shell in docker enviroment)
> If so, it will be helpful for rendering tests.

Yes, I am trying to run the test shell in the `build-engine` docker image. But I am currently facing some errors.. 😭

**Comment by WonyoungChoi** [comment]
Can someone approve of this? This must be merged first for the next task.

**Review by bbrto21 — APPROVED** [review]
LGTM

**Comment by WonyoungChoi** [comment]
> LGTM

Thank you so much! 😺

File: Dockerfile [hunk 2]
```diff
@@ -35,18 +35,18 @@ RUN mkdir -p ${SYSROOT_PATH}
...
-        b=$(basename $f); \
-        cp $f ${TOOLCHAINS_PATH}/bin/aarch64-tizen-${b:8}; \
+    b=$(basename $f); \
+    cp $f ${TOOLCHAINS_PATH}/bin/aarch64-tizen-${b:8}; \
     done
 RUN for f in ${TIZEN_STUDIO}/tools/i586-linux-gnueabi-gcc-9.2/bin/i586-linux-*; do \
-        b=$(basename $f); \
-        cp $f ${TOOLCHAINS_PATH}/bin/i586-tizen-${b:5}; \
+    b=$(basename $f); \
+    cp $f ${TOOLCHAINS_PATH}/bin/i586-tizen-${b:5}; \
```
[the remark this answers is not in the document]
Comment (swift-kim) [reply Dockerfile:49]: Hello? Is this merged?

[the remark this answers is not in the document]
Comment (WonyoungChoi) [reply Dockerfile:49]: oops.. my vscode formatter makes this changes. I will fix it later in other PR. 
I merged this PR early for the dependency of https://github.com/flutter-tizen/engine/pull/181.

**Pull request merged**